### PR TITLE
Grouped Dependabot Changes

### PR DESCRIPTION
**Pull request by GrantBirki**
related: https://github.com/the-hideout/tarkov-dev/pull/796

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,14 +1,21 @@
 # Please see the documentation for all configuration options:
 # https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
-
+---
 version: 2
 updates:
-  - package-ecosystem: "npm"
-    directory: "./src/tarkov-data-manager"
+  - package-ecosystem: github-actions
+    directory: "/"
```
Comment (GrantBirki) [inline .github/dependabot.yml:7]:
```suggestion
    directory: "./src/tarkov-data-manager"
```

**Review by Shebuka — APPROVED** [review]

**Comment by GrantBirki** [comment]
.deploy

**Comment by GrantBirki** [comment]
.deploy

**Comment by GrantBirki** [comment]
.wcid

**Comment by GrantBirki** [comment]
I'm going to deploy/merge these changes in. From there I will re-deploy the currently deploy PR -> https://github.com/the-hideout/tarkov-data-manager/pull/475 so that is up-to-date again

**Comment by GrantBirki** [comment]
.unlock

**Comment by GrantBirki** [comment]
.deploy

**Pull request merged**